ui/ozone/platform/wayland/wayland_connection.h

```diff
@@ -68,6 +73,20 @@ class WaylandConnection : public PlatformEventSource,
   // not delivered.
   void ResetPointerFlags();
 
+  // Clipboard implementation.
```

Comment:
I wonder if this can be moved to some other object to avoid overloading the WaylandConnection that much.

Reply:
as we know, connection is the class that instantiates data_offer, data_device, data_device_manager, and data_source objects, similarly to how it does with other wl objects like pointer, keyboard and touch.

it just happens that wayland's design of putting wl_display as a central object causes the overload of its wrappers classes (eg wayland connection).

we might be able to factor it out, in the future, yes.